### ISSUE TITLE
Wrong `std` value in CIFAR10 config 

### ISSUE THREAD

**Issue by nps1ngh** [chenyaofo/image-classification-codebase#5]
Hi,

the value of `std` in the config file: 
https://github.com/chenyaofo/image-classification-codebase/blob/19a5ce630de8d2ab29f0a63ff585a10f92335273/conf/cifar10.conf#L22

seems to be wrong. It should be `[0.2470, 0.2435, 0.2616]` instead. 

See [this comment](https://gist.github.com/weiaicunzai/e623931921efefd4c331622c344d8151?permalink_comment_id=2851662#gistcomment-2851662).

Some code to calculate it:
```python
import torchvision

c10 = torchvision.datasets.CIFAR10(
    ".",
    train=True,
    transform=torchvision.transforms.ToTensor(),
)

all_images = torch.stack([t[0] for t in c10])  # .shape == (50_000, 3, 32, 32)
print("mean", all_images.mean(dim=(0, 2, 3)))  # [0.4914, 0.4822, 0.4465]
print("std", all_images.std(dim=(0, 2, 3)))    # [0.2470, 0.2435, 0.2616]
```

**Comment by chenyaofo** [comment]
Thanks for your feedback. I need some time to comfirm it.

**Comment by nps1ngh** [comment]
Seems like the std values were computed by averaging the individual image std values (I was able to reproduce them that way).
In other words, they were calculated using:
```python
all_images.std(dim=(2,3)).mean(dim=0)
```
instead of 
```python
all_images.std(dim=(0, 2, 3))
```

This is also how the normalization constants in ImageNet came about (see https://github.com/pytorch/vision/pull/1965).
Although, I'd argue this is not exactly calculating the standard deviation over the entire dataset (as pointed out [here](https://discuss.pytorch.org/t/computing-the-mean-and-std-of-dataset/34949/5) and also how the torchvision issue referred above came about), I agree with the comment made [here](https://github.com/pytorch/vision/issues/1439#issuecomment-544713280) and think that retraining the models here (for CIFAR10) also would not make sense.

As such I'll close this issue.
